sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryServicesImplTest.java

```diff
@@ -498,7 +498,7 @@ public void testInsertRetry() throws Exception {
 
```

Comment:
This is the file I'd expect to see a test that the `ignoreUnknownValues` field actually makes it to the `TableDataInsertAll` request. Maybe @peihe can suggest how such a test could be written?